File: spring-ceph-client/src/test/java/com/example/HelloTests.java
package com.example;

import org.junit.jupiter.api.Test;

import lombok.extern.slf4j.Slf4j;

@Slf4j
public class HelloTests extends EvaluatedTimeTests {

    @Test
    public void hello() {
        System.out.println("hello");
        log.info("hello");
    }
}

File: spring-httpclient-kotlin/src/test/kotlin/com/example/HelloTests.kt
package com.example

import io.github.oshai.kotlinlogging.KotlinLogging
import org.junit.jupiter.api.Test

class HelloTests : EvaluatedTimeTests() {
    private val log = KotlinLogging.logger {}

    @Test
    fun hello() {
        println("hello")
        log.info { "hello" }
    }
}
